### PR TITLE
add continuables.jl for loading paths

### PR DESCRIPTION
**Pull request by ReneSkukies**
Loading the paths was a bottle neck so far, this should make file lookup much faster. Additionally, that fixes a bug during loading and makes the code much more readable

### CODE REVIEW

File: src/load.jl [hunk 1]
```diff
@@ -1,85 +1,71 @@
...
-    if derivative && excludeFolder !== nothing
-        exclude = excludeFolder
-    elseif !derivative && excludeFolder !== nothing
-        exclude = ["derivatives", excludeFolder]
-    elseif !derivative
-        exclude = "derivatives"
+    # Choose what to ignore and check if derivatives should be used
+    exclude = [] 
+    if derivatives
+        # TODO: Bug in loading only derivatives
```
Comment (behinger) [inline src/load.jl:29]: what bug?

File: src/load.jl [hunk 2]
```diff
@@ -1,85 +1,71 @@
...
-                if sum(occursin.(file_pattern, file)) >= nPattern &&
-                   (derivative && (exclude == "" || !any(occursin.(exclude, root))) ||
-                    (!derivative && !any(occursin.(exclude, root))))
-
-                    get_info!(files_df, root, file)
-                end
+            for file in readdir(path)
+                foreach(cont, list_all_eegpaths(joinpath(path, file)))
             end
         end
```
Comment (behinger) [inline src/load.jl:59]: I would add an `else @throw "file not found $path"`

Comment (ReneSkukies) [reply src/load.jl:58]: Changed this so there is an error if no files at all are found.

File: src/load.jl (hunk 1 above)
Comment (ReneSkukies) [reply src/load.jl]: Actually already fixed the bug, just forgot to delete the TODO...

**Comment by behinger** [comment]
ready to merge !

giphy

**Pull request merged**